my_enumerable.rb

```diff
@@ -0,0 +1,20 @@
+module MyEnumerable
+  def all?()
+    each do |item|
+      return false unless yield(item)
+    end
+    true
+  end
+
+  def any?()
+    each do |item|
+      return true if yield(item)
+    end
+    false
+  end
+
+  def filter(&block)
+    select(&block)
+    # end
+  end
```

Comment:
- [ ] Kindly consider using the newly created `each` method here instead of the default `each` method